pkgs/development/compilers/llvm/rocm/default.nix

```diff
@@ -0,0 +1,40 @@
+{ stdenv, fetchFromGitHub, callPackage, wrapCCWith }:
+
+let
+  version = "3.5.1";
+  src = fetchFromGitHub {
+    owner = "RadeonOpenCompute";
+    repo = "llvm-project";
+    rev = "rocm-${version}";
+    sha256 = "03k2xp8wf4awf1zcjc2hb3kf9bqp567c3s569gp1q3q1zjg6r2ib";
+  };
+in rec {
+  clang = wrapCCWith rec {
+    cc = clang-unwrapped;
+    extraBuildCommands = ''
+      rsrc="$out/resource-root"
+      mkdir "$rsrc"
+      ln -s "${cc}/lib/clang/11.0.0/include" "$rsrc"
```

Comment:
Can we automate the version number part of this path? It is a nuisance to maintain because we get version bumps independent of public releases, requiring that a person make sure this line is in sync with wherever the rocm fork is with respect to upstream's development branch.

Reply:
Changed this to get the version from `clang` itself.